app/static/openapi.yaml

```diff
@@ -253,7 +253,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Category_data'
+                $ref: '#/components/schemas/CategoryDataList'
```

Comment:
`GET /category/<id>` doesn't actually return a list, so I'm not sure why it would reference a schema called `CategoryDataList` now... I'm reluctant to continue reviewing this PR because the first one appears to make it worse rather than make it better.

Can you please post some screenshots of the before and after so we can see how this changes the way the docs are rendered at `http://localhost:5000`?

Reply:
okay sure

Reply:
@aaron-suarez I did check again and it seems there isn't any schema that matches /categories/{id}